frontend/app/model/ResponsiveImage.scala

```diff
@@ -7,7 +7,7 @@ case class ResponsiveImageGroup(
   altText: String,
   availableImages: Seq[ResponsiveImage]
 ) {
-  val defaultImage = availableImages.head.path
+  val defaultImage = availableImages.headOption.map(_.path)
```

Comment:
I am finding that sometimes the list of availableImages is sometimes empty hence changing this to an option